packages/plugin-essentials/sources/commands/exec.ts

```diff
@@ -0,0 +1,32 @@
+import {BaseCommand}            from '@yarnpkg/cli';
+import {Configuration, Project} from '@yarnpkg/core';
+import {execUtils, scriptUtils} from '@yarnpkg/core';
+import {xfs}                    from '@yarnpkg/fslib';
+import {Command}                from 'clipanion';
+
+// eslint-disable-next-line arca/no-default-export
+export default class ExecCommand extends BaseCommand {
+  @Command.String()
+  commandName!: string;
+
+  @Command.Proxy()
+  args: Array<string> = [];
+
```

Comment:
Sounds good - can you add a usage definition? Otherwise it won't show up in the website documentation.

Reply:
Done. 👍